Tribler/community/allchannel/__init__.py

```diff
@@ -0,0 +1,4 @@
+'''
+The allchannel package contains contains code for a single community 
+that all Tribler members join and use to disseminate .torrent files.
```

Comment:
Not true, the AllChannel community is used to collect votes for channels, and thereby discover which channels are most popular.